Source/Workers/createPointGeometry.js

```diff
@@ -0,0 +1,6 @@
+/*global define*/
+define(['../Core/PointGeometry'], function(PointGeometry) {
```

Comment:
Can you run the `sortRequires` build step (after merging in master) so that these headers get formatted properly.